### DISCUSSION TITLE
Readable theme

### DISCUSSION THREAD

**Discussion by alfonz19** [dbeaver/dbeaver#11499] [Ideas]
Idea of having multiple themes users can pick from is fine. But would it be actually possible to provide at least one setting, where EVERYTHING is readable on screen? I tested every provided setting, and there is always something which is invisible or I need to hover over to make it visible. Having gradients in panel is fancy and might be selling point to someone, but IIUC this should be tool for work and not being able to read buttons easily in default setting is problem. 

For me disabling theming and mixed colors makes everything readable, with some problems, but readable. Can you reconsider creating new theme, "easy to read" (like black on white instead light gray over white etc), focused on fast work instead of looks?

EDIT: my bad, I was able to made panels readable only to find out, that I'm now unable to read the main window into which I type sql statements. It might be some conflict with system theme, I don't know, but I'd like to achieve the same what is possible with intellij idea: regardless of anything, if you run it with default settings, everything is readable.

If it's not possible to do this, please suggest how to change background color of panels and main edit pane.

Thanks.

**Comment by kseniiaguzeeva** [comment]
Try Classic theme, because it has optimal colors. If some items still have bad visibility, please tell which ones, your OS and DBeaver version

**Comment by pgodfrin-nov** [comment]
Any thoughts about a Solarized theme (dark too)?

**Comment by kseniiaguzeeva** [comment]
@pgodfrin-nov  please clarify what do you mean.

**Comment by pgodfrin-nov** [comment]
It would be nice to have the Solarized themes, or even any of the dozens that are out there for other products.

**Comment by kseniiaguzeeva** [comment]
We are not going to implement new themes, but we support eclipse plugin https://marketplace.eclipse.org/content/eclipse-color-theme. Please try it.

**Comment by flatroof** [comment]
New user here.. insane amount of tiny buttons with no context. I really like the idea of this tool but is there a way to make the UI usable?
